sw/device/examples/sram_program/sram_program.c

```diff
@@ -22,6 +24,11 @@ enum {
 };
 
 void sram_main() {
+  // Configure the pinmux.
```

Comment:
Ah, you need this because you are breaking at the start of the `rom_test_main()` before pinmux is initialized.